### PR TITLE
Button,rating,image,input

### PR DESCRIPTION
**Pull request by rov9**
Button,rating,image,input components,please review

### CODE REVIEW

File: componenets/getstarted.html [hunk 1]
```diff
@@ -18,26 +18,31 @@
         <a href="#">Alert</a>
         <a href="/componenets/avatar.html">Avatar</a>
```
Comment (srejitk) [inline componenets/getstarted.html:19]: Check the spelling of the component folder

File: xpressUI.css [hunk 2]
```diff
@@ -32,7 +33,7 @@
   .side-nav a {
     padding: 8px 8px 8px 32px;
     text-decoration: none;
-    font-size: 20px;
+    font-size: 17px;
```
Comment (srejitk) [inline xpressUI.css]: Use relative units instead of px

File: xpressUI.css [hunk 3]
```diff
@@ -98,3 +104,124 @@
...
+
+  /*------------------Button------------------*/
+
+  .button{
+
+    margin: 1rem;
+    padding: 1rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```
Comment (krishanu2911) [inline xpressUI.css:130]: just a suggestion that you can move this common property to utils

File: xpressUI.css [hunk 4]
```diff
@@ -98,3 +104,124 @@
...
+
+
+  .link-btn{
+      border:none;
+      background-color: var(--white-background);
+  }
+
+  .icon-btn{
+    display: flex;
+    align-items: center;
```
Comment (krishanu2911) [inline xpressUI.css:155]: same here

File: xpressUI.css [hunk 5]
```diff
@@ -98,3 +104,124 @@
...
+    height:50px;
+    border-radius: 100%;
+    background: var(--background-color);
+    box-shadow: 10px 10px 5px #aaaaaa;
+    }
+
+    .float-btn i{
+        display:flex;
+        align-items:center;
+        justify-content:center;
```
Comment (krishanu2911) [inline xpressUI.css:170]: these properties too

**Review by krishanu2911 — APPROVED** [review]
Everything is good to go, Just added few suggestions

File: xpressUI.css (hunk 3 above)
Comment (srejitk) [inline xpressUI.css:130]: All these properties can be extracted into a utility class

File: xpressUI.css (hunk 5 above)
Comment (srejitk) [inline xpressUI.css:170]: Extract this into a utility class

**Review by srejitk — APPROVED** [review]
Suggested some minor changes. Put all the respective styles in the respective component css

File: componenets/button.html [hunk 6]
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Button</title>
```
Comment (kjais1720) [inline componenets/button.html:7]: You can put your CL name as well, along with the page title

File: componenets/button.html [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+    <link rel="stylesheet" href="/xpressUI.css">
+    <script src="https://kit.fontawesome.com/2ea00fb38c.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div id="mySidenav" class="side-nav">
+        <a href="javascript:void(0)" class="closebtn" id="close-sidebar">&times;</a>
```
Comment (kjais1720) [inline componenets/button.html:13]: If this is a button then you should use the `<button>` tag instead of the anchor tags

File: index.html [hunk 8]
```diff
@@ -21,10 +21,10 @@
...
-        <a href="/components/getstarted.html"><button 
+        <a href="/componenets/getstarted.html"><button 
         class="landing-content btn-index">Get Started</button></a>
-        <a href="https://github.com/rov9/xPress.Ui"><button 
-        class="landing-content btn-index">Github</button></a>
+        <a href="https://myblog4you.hashnode.dev/documentation"><button 
+        class="landing-content btn-index">Documentation</button></a>
         </div>
     
 </body>
```
Comment (kjais1720) [inline index.html]: You should use Prettier or any code formatter to make the code properly indented, it's not very readable right now

**Review by kjais1720 — APPROVED** [review]
Just a couple of suggestion, if you have time then fix them now or you can refactor it later, the code looks good overall👍

**Pull request merged**